test_requirements.txt

```diff
@@ -25,7 +25,7 @@ first==2.0.1              # via pip-tools
 greenlet==0.4.11          # via bpython
 ipdb==0.10.1
 ipython-genutils==0.1.0   # via traitlets
-ipython==5.1.0
+ipython==5.1.0            # via ipdb
```

Comment:
ipdb is a debugger: I do not see the reason to move it. 
maybe I can keep ipython here with a strict requirement. I'll test something